static/app/components/onboardingWizard/filterSupportedTasks.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  feedbackOnboardingPlatforms,
+  replayOnboardingPlatforms,
+  withoutPerformanceSupport,
+} from 'sentry/data/platformCategories';
+import {type OnboardingTask, OnboardingTaskKey, type Project} from 'sentry/types';
+
+const replayRelatedTasks = [OnboardingTaskKey.SESSION_REPLAY];
+const performanceRelatedTasks = [
+  OnboardingTaskKey.FIRST_TRANSACTION,
+  OnboardingTaskKey.PERFORMANCE_GUIDE,
+  OnboardingTaskKey.METRIC_ALERT,
+];
+const feedbackRelatedTasks = [OnboardingTaskKey.USER_REPORTS];
+
+export function filterSupportedTasks(
+  projects: Project[] | undefined,
+  allTasks: OnboardingTask[]
+): OnboardingTask[] {
+  if (!projects) {
+    return [];
+  }
+  // Remove tasks for features that are not supported
+  return allTasks.filter(
+    task =>
+      (shouldShowReplayTasks(projects) || !replayRelatedTasks.includes(task.task)) &&
+      (shouldShowPerformanceTasks(projects) ||
+        !performanceRelatedTasks.includes(task.task)) &&
+      (shouldShowFeedbackTasks(projects) || !feedbackRelatedTasks.includes(task.task))
+  );
+}
+
+export function shouldShowPerformanceTasks(projects: Project[]): boolean {
+  return !projects?.every(
```

Comment:
just confirming this is supposed to be a not?

Reply:
Yes, so as you see the feedback one and replay one don't have ! but they're `some`. Performance is different because instead of PerfromancePlatforms what we have in platform categories is withoutPerformanceSupport platforms.

So basically:
Do `NOT` show performance tasks if `every` project for this org is `withoutPerformanceSupport`.